index.bs

```diff
@@ -70,32 +87,27 @@ however, including (but not limited to):
 
 These use-cases have several common properties:
 
-1. They can be represented as passive "queries"
-	about the state of individual elements
-	with respect to some other element
-	(or the global viewport)
-2. They do not impose hard latency requirements;
-	that is to say, the information can be delivered asynchronously
-	(e.g. from another thread)
-	without penalty
-3. They are poorly supported by nearly all combinations of existing web platform features,
-	requiring extraordinary developer effort despite their widespread use.
-
-A notable non-goal is pixel-accurate information about what was actually displayed
-(which can be quite difficult to obtain efficiently in certain browser architectures
-in the face of filters, webgl, and other features).
-In all of these scenarios the information is useful
-even when delivered at a slight delay
-and without perfect compositing-result data.
-
-The Intersersection Observer API addresses the above issues
-by giving developers a new method to asynchronously query the position of an element
-with respect to other elements or the global viewport.
-The asynchronous delivery eliminates the need for costly DOM and style queries,
-continuous polling,
-and use of custom plugins.
-By removing the need for these methods
-it allows applications to significantly reduce their CPU, GPU and energy costs.
+1. They can be represented as passive "queries" about the state of individual
+	elements with respect to some other element (or the global viewport).
+2. They do not impose hard latency requirements; that is to say, the
+	information can be delivered asynchronously (e.g. from another thread)
+	without penalty.
+3. They are poorly supported by nearly all combinations of existing web platform
+	features, requiring extraordinary developer effort despite their
+	widespread use.
+
+A notable non-goal is pixel-accurate information about what was actually
+displayed (which can be quite difficult to obtain efficiently in certain
+browser architectures in the face of filters, webgl, and other features). In all
+of these scenarios the information is useful even when delivered at a slight
+delay and without perfect compositing-result data.
+
+The Intersersection Observer API addresses the above issues by giving developers
+a new method to asynchronously query the position of an element with respect to
+other elements or the global viewport. The asynchronous delivery eliminates the
+need for costly DOM and style queries, continuous polling, and use of custom
+plugins. By removing the need for these methods it allows applications to
+significantly reduce their CPU, GPU and energy costs.
 
```

Comment:
I can't tell what you did or didn't change here.  When I generally reformatted the spec I switched to [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/), which I use in all of my specs and which make diffs better.

If you didn't actually change anything, can you revert this back to the formatting it had?